requirements/edx/development.txt

```diff
@@ -316,9 +332,7 @@ django-cache-memoize==0.1.10
     #   -r requirements/edx/testing.txt
     #   edx-enterprise
 django-celery-results==2.0.1
```

Comment:
You need to run make upgrade again, so the latest version of `django-celery-results` is fetched as the constraint is removed from this package